src/PrestaShopBundle/Form/Admin/Sell/Order/OrderPaymentType.php

```diff
@@ -110,14 +110,6 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 'required' => false,
             ])
             ->add('amount', NumberType::class, [
-                'constraints' => [
```

Comment:
Even if you add a constraint directly on the database definition, I think it's better to let this check at this point. Because with this check, the error can be handled and shown to the user directly on the form

Reply:
Sorry, I understood the opposite... I resolve my review, waiting for PM return.

Reply:
What is PM Return? What am I supposed to do now?